engine/access/rpc/backend/backend_accounts.go

```diff
@@ -41,6 +42,7 @@ func (b *backendAccounts) GetAccountAtLatestBlock(ctx context.Context, address f
 
 	account, err := b.getAccountAtBlockID(ctx, address, latestBlockID)
 	if err != nil {
+		b.log.Error().Stack().Err(err).Msg(fmt.Sprintf("failed to get account at blockID: %v", latestBlockID))
```

Comment:
why the `.Stack()` ?

Reply:
to auto-add formatted stack trace, will that be too verbose?

Reply:
yeah probably since we do get a lot of errors